app/models/dog.rb

```diff
@@ -7,7 +7,7 @@ class Dog < ActiveRecord::Base
   has_many :views, class_name: "Observation", foreign_key: "observer_id"
   has_many :viewed_by,  class_name:  "Observation", foreign_key: "observed_id"
   has_attached_file :img, storage: :s3, bucket: ENV['BUCKET_NAME'],
-                          default_url: "puppy1.jpeg"
+                    default_url: "puppy1.jpeg", styles: { :original =>"200x200#",  }
```

Comment:
Line is too long. [84/80]<br>Align the elements of a hash literal if they span more than one line.<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.